internal/runtime/init/init.go

```diff
@@ -127,10 +127,15 @@ func LoadPaths(paths []string,
 		caps = ast.CapabilitiesForThisVersion()
 	}
 
+	var targzs, nonTargzs []string
```

Comment:
suggestion: I think it'd be clearer to use variable names that hint at what's going on here, we're trying to make a guess that tar.gz files are bundles.

```suggestion
	var likelyBundles, nonBundlePaths []string
```